plugins/woocommerce/client/admin/client/marketing/components/PluginCardBody/SmartPluginCardBody.tsx

```diff
@@ -2,12 +2,12 @@
  * External dependencies
  */
 import { useState } from '@wordpress/element';
-import { useDispatch } from '@wordpress/data';
+import { useDispatch, useSelect } from '@wordpress/data';
 import { Button } from '@wordpress/components';
 import { Pill } from '@woocommerce/components';
 import { __ } from '@wordpress/i18n';
 import { recordEvent } from '@woocommerce/tracks';
-import { PLUGINS_STORE_NAME } from '@woocommerce/data';
+import { PLUGINS_STORE_NAME, PluginSelectors } from '@woocommerce/data';
```

Comment:
we could explicitly import the type
```suggestion
import { PLUGINS_STORE_NAME, type PluginSelectors } from '@woocommerce/data';
```

Reply:
Done in [`82bbd9e` (#54485)](https://github.com/woocommerce/woocommerce/pull/54485/commits/82bbd9e5d1bb5324d16e620aec840ade52d03c0c).